src/components/action-popover/action-popover-item.component.js

```diff
@@ -261,7 +273,11 @@ const propTypes = {
   /** The name of the icon to display next to the label */
   icon: PropTypes.oneOf(OptionsHelper.icons),
   /** Callback to run when item is clicked */
-  onClick: PropTypes.func.isRequired,
+  onClick: PropTypes.func,
```

Comment:
Why is this no longer required? 
If there's a good reason for it, you need to change the TypeScript interface as well

Reply:
See my above comment, I think this needs to stay as required